backend/src/gpml/db/browse.clj

```diff
@@ -45,4 +45,8 @@
 
   (count (filter-topic db {:search-text "closed" :topic ["technical_resource"]}))
 
+  (count (filter-topic db {:tag ["litter monitoring"]}))
+
+  (topic-counts db {:tag ["litter monitoring"]})
```

Comment:
perhaps we should move this dev logic to dev folder

Reply:
Or, we could just get rid of the whole comment section from this file.

Reply:
it's up to you 😄 !